### PR TITLE
Job descriptions

### PR DESCRIPTION
**Pull request by mhefferan**
Blog about yesterdays activities

### CODE REVIEW

File: posts/job_descriptions.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Like most small companies we have trouble [@MyDrive](https://twitter.com/_mydrive) in attracting the best
+engineering staff when we're recruiting. We're not Google or Apple, with a huge
+global presence, most developers have probably never heard of us. The current
+job market for developers is really competitive, there aren't that many great
+developers out there, most of them are already in work and everyone is competing
+for them.
+
+We've recently been recruiting for an iOS developer role, to lead the development
+of our telematics apps. I think is actually a really interesting, fun role, which
```
Comment (jfharden) [inline posts/job_descriptions.md]: typo: I think is actually -> I think it's actually

**Comment by jfharden** [comment]
Other than the minor typo :ship: :it:

File: posts/job_descriptions.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+We got two great lists, and decided to vote on which of the things we'd got on
+our list were the most important, so we could focus on those in our adverts.
+
+![Results](/assets/media/job_voting/results.jpg)
+
+The biggest winner was Remote Working Encouraged, which is really interesting. My
+offer letter contained the phrase "some remote working encouraged", and I remember
+thinking at the time that that statement was great. It's not allowed, or tolerated,
+there is an understanding that some remote working is actually beneficial, both to
```
Comment (willfish) [inline posts/job_descriptions.md]: Do you mean to say: it's not allowed or tolerated in most companies, and there is an understanding that some remote working is actually beneficial...

File: posts/job_descriptions.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Some recruiters have told us that people are put off from applying because there
+is a more challenging process including a test, and hopefully that's just filtering
+out people who aren't suitable for the role, but we could be putting off some
+candidates who really would be great because we've increased the barrier to apply.
+
+All of this caused us to review our Job Descriptions. We quickly realised
+that just tweaking wording maybe wasn't radical enough. There was also a feeling
+that one of the best things about working at [@MyDrive](https://twitter.com/_mydrive) is the working culture
+which wasn't well reflected in the job descriptions.
```
Comment (willfish) [inline posts/job_descriptions.md]: Should this be: "which wasn't well reflected in the job description." without the 's'

File: posts/job_descriptions.md (hunk 2 above)
Comment (mhefferan) [inline posts/job_descriptions.md]: hmm. ok so I'm trying to get across that it isn't merely allowed or tolerated, it's actually phrased as encourages. I'll reword

**Comment by willfish** [comment]
:ship: :it:

File: posts/job_descriptions.md (hunk 3 above)
Comment (mhefferan) [inline posts/job_descriptions.md]: I start the paragraph mentioning our job descriptions, then re-reference them, but the capitalisation changes so I'll modify that

File: posts/job_descriptions.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+our list were the most important, so we could focus on those in our adverts.
+
+![Results](/assets/media/job_voting/results.jpg)
+
+The biggest winner was Remote Working Encouraged, which is really interesting. My
+offer letter contained the phrase "some remote working encouraged", and I remember
+thinking at the time that that statement was great. It's not described as allowed, 
+or merely tolerated at MyDrive it's encouraged. There is an understanding that some
+remote working is actually beneficial, both to individuals and to the company, and
+the tools are in place to make remote working easy.
```
Comment (gavinheavyside) [inline posts/job_descriptions.md]: I'd like to rephrase this paragraph. I don't know that "encouraged" is the right verb, "actively supported" might be more appropriate; working from home is subject to management consent, and isn't a contractual right. So long as it doesn't negatively impact the business (indeed it usually positively impacts) then we'll be extremely accommodating compared with most companies.

Comment (mhefferan) [inline posts/job_descriptions.md]: encouraged is what you put in my offer letter! :D I'll rephrase

File: posts/job_descriptions.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+![Categories](/assets/media/job_voting/categories.jpg)
+
+
+We got two great lists, and decided to vote on which of the things we'd got on
+our list were the most important, so we could focus on those in our adverts.
+
+![Results](/assets/media/job_voting/results.jpg)
+
+The biggest winner was about the Remote Working policy, which is really interesting.
```
Comment (mhefferan) [inline posts/job_descriptions.md]: @gavinheavyside is this better?

Comment (gavinheavyside) [inline posts/job_descriptions.md]: Yes, and I think this probably highlights that we need a written remote working policy to make sure everyone is absolutely clear on what they can expect, and what is expected.

**Comment by mhefferan** [comment]
So can I ship?

**Comment by gavinheavyside** [comment]
:shipit:

**Pull request merged**